### ISSUE TITLE
Add files to git ignore

### ISSUE THREAD

**Issue by Pajicc** [tamiat/tamiat#110]
We should put config.js file to git ignore. If you have any suggestions on what else should we land there, please write it

**Comment by Abdelaziz18003** [comment]
config.js don't contains just credentials .. it contains also firebaseRefs .. I think we should separate them before adding config.js to gitignore because databaseRefs should not be gitignored .. or create an other file just for credentials and don't track it with git ... @mahnouman what do you think ?

**Comment by mornir** [comment]
Yes, I think it is best to have a separate file that only contains the Firebase config and to add it to gitignore.

**Comment by mahnouman** [comment]
yes, that is a good idea, let us have a file with just firebase config and add this into the gitignore. Any other files or folders should be added to gitignore as well?
